pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>uk.gov.companieshouse</groupId>
   <artifactId>company-accounts.api.ch.gov.uk</artifactId>
-  <version>unversioned</version>
+  <version>5cb2b632</version>
```

Comment:
Should this be a hash or unversioned?

Reply:
I didn't intend to commit that actually.  I'll set it back..